### PR TITLE
Creates a healthy alternative for breakfast!

### PR DESCRIPTION
**Pull request by Wallemations**
## About The Pull Request

Creates a delicious breakfast with just 650 calories! Perfect for anyone hoping for a delicious meal!

## Why It's Good For The Game
healthy_diet

## Changelog
:cl: Wallem
add: Upon Nanotrasen's dietitians releasing their recommended protein intake data, a new egg-based meal has been added to the station's menus!
/:cl:

### CODE REVIEW

**Comment by Jolly-66** [comment]
currently working on an upstream merge.
melbert don't review this yet until the repo is updated

**Comment by MrMelbert** [comment]
image
reeeeeeee
https://github.com/Jolly-66/JollyStation/blob/master/jollystation_modules/README.md

**Comment by Jolly-66** [comment]
also i wanna see the sprites, so whenever you can either comment them or put them in your main pr

File: code/game/objects/items/food/egg.dm [hunk 1]
```diff
@@ -137,3 +137,14 @@
 	w_class = WEIGHT_CLASS_NORMAL
 	tastes = list("egg" = 1, "bacon" = 1, "bun" = 1)
 	foodtypes = MEAT | BREAKFAST | GRAIN
+
+/obj/item/food/balancedegg //13140 eggs per year
+	name = "balanced breakfast"
+	desc = "Only 650 calories, perfect for any dieting station members!"
+	icon_state = "balancedegg"
+	trash_type = /obj/item/trash/plate
+	list_reagents = list(/datum/reagent/consumable/nutriment/protein = 108, /datum/reagent/consumable/nutriment/vitamin = 36) //520 kcals from protein in a diet of 650 kcals
```
Comment (Jolly-66) [inline code/game/objects/items/food/egg.dm:146]:
```suggestion
/obj/item/food/balancedegg
	name = "balanced breakfast"
	desc = "Only 650 calories, perfect for any dieting station members!"
	icon_state = "balancedegg"
	trash_type = /obj/item/trash/plate
	list_reagents = list(/datum/reagent/consumable/nutriment/protein = 108, /datum/reagent/consumable/nutriment/vitamin = 36)
```

don't need these comments, they're just a meme only the 3 of us will see

File: code/modules/food_and_drinks/recipes/tablecraft/recipes_egg.dm [hunk 2]
```diff
@@ -51,3 +51,12 @@
 	)
 	result = /obj/item/food/salad/eggbowl
 	subcategory = CAT_EGG
+
+/datum/crafting_recipe/food/balancedegg
+	name= "Balanced Breakfast"
+	reqs = list(
+		/obj/item/reagent_containers/food/snacks/boiledegg = 36, //SPECIFICALLY Hard boiled eggs
```
Comment (Jolly-66) [inline code/modules/food_and_drinks/recipes/tablecraft/recipes_egg.dm:58]:
```suggestion
		/obj/item/reagent_containers/food/snacks/boiledegg = 36, 
```

again, meme comment no one else will see

**Review by Jolly-66 — CHANGES_REQUESTED** [review]
> image
> reeeeeeee
> https://github.com/Jolly-66/JollyStation/blob/master/jollystation_modules/README.md

melbert did it before i felt like it
modularize the dmi

**Comment by Wallemations** [comment]
> > image
> > reeeeeeee
> > https://github.com/Jolly-66/JollyStation/blob/master/jollystation_modules/README.md
> 
> melbert did it before i felt like it
> modularize the dmi

Ah. Fun. I'll remember to do that.

**Comment by Jolly-66** [comment]
since melbert converted this to a draft, the merge conflicts are hidden, but they were all 3 files

**Comment by Jolly-66** [comment]
also i just want to say i'm personally not as huge fan of the sprite, it just looks like a ball of eggs
i get this is mostly a joke, but i highly doubt someone would go out of there way to try and make this, considering it needs a metric fuckton of eggs that could be put elsewhere, and would require a botanist to grow egg-plants to help offset the now missing eggs

**Comment by Jolly-66** [comment]
wallems going to come back with a different idea

**Pull request closed without merging**